adapter/integration-test/src/adapter-datasource.test.js

```diff
@@ -184,6 +187,52 @@ describe('Adapter Configuration', () => {
   })
 })
 
+test('POST datasources/{id}/trigger', async () => {
```

Comment:
Does it make sense to split dynamic and non-dynamic into separate test cases?

Reply:
Maybe, but actually it should be explicit for the trigger endpoint. I followed the other tests that test for the endpoint as well.

Reply:
The reasoning is that all data sources are treated as dynamic anyway. Either they have open parameters or not. But if you think that makes more sense, then it is fine with me.